src/release.py

```diff
@@ -477,48 +478,22 @@ def build_blog_post(repositories, temp_dir, token):
 def generate_new_tag(repository, temp_dir, specified_crate):
     versions = {}
     version = None
-    # In some repositories (like sys), there are more than one crates. In such case, we try to
-    # get the most common version number and then we create the tag from there.
+
+    # We make a new tag for every crate:
     #
-    # First, we get all versions.
+    # * If it is a "sys" crate, then we add its name to the tag
+    # * If not, then we just keep its version number
     for crate in consts.CRATE_LIST:
         if crate['repository'] == repository:
-            versions[crate['crate']] = CRATES_VERSION[crate['crate']]
+            tag_name = CRATES_VERSION[crate['crate']]
             if crate['crate'].endswith('-sys') or crate['crate'].endswith('-sys-rs'):
-                version = CRATES_VERSION[crate['crate']]
-    if (specified_crate is not None and
-            (specified_crate.endswith('-sys') or specified_crate.endswith('-sys-rs'))):
-        write_msg('Seems like "{}" is part of a repository with multiple crates so no \
-                   tag generation this time...'.format(specified_crate))
-        return
-
-    if version is None:
-        most_common = {}
-        # Now we get how many crates have this version.
-        for version in versions:
-            if versions[version] in most_common:
-                most_common[versions[version]] += 1
-            else:
-                most_common[versions[version]] = 1
-        # Now we get the "lowest" version that will be used as default tag name.
-        for common in most_common:
-            if version is None or compare_versions(common, version) < 0:
-                version = common
-        # And now we get the most common tag name.
-        for common in most_common:
-            if version is None or most_common[version] < most_common[common]:
-                version = common
-        if version is None:
-            write_error('Something impossible happened for "{}": no version can be tagged...'
-                        .format(repository))
-            input('If you think you can do better, go ahead! (In "{}") Then press ENTER to continue'
-                  .format(join(temp_dir, repository)))
-            return
-    write_msg('==> Creating new tag "{}" for repository "{}"...'.format(version, repository))
-    create_tag_and_push(version, repository, temp_dir)
+                tag_name = '{}-{}'.format(crate['crate'], CRATES_VERSION[crate['crate']])
```

Comment:
why not `tag_name = '{}-{}'.format(crate['crate'], tag_name)` ?

Reply:
:wink:

Reply:
But thanks in any case, I'll update this evening. :smiley:

Reply:
It was good as is it just unneeded nit

Reply:
I think your suggestion is better than the current code so I'll follow it.